include/deal.II/lac/solver_idr.h

```diff
@@ -0,0 +1,479 @@
+// ---------------------------------------------------------------------
+//
+// Copyright (C) 2000 - 2019 by the deal.II authors
+//
+// This file is part of the deal.II library.
+//
+// The deal.II library is free software; you can use it, redistribute
+// it, and/or modify it under the terms of the GNU Lesser General
+// Public License as published by the Free Software Foundation; either
+// version 2.1 of the License, or (at your option) any later version.
+// The full text of the license can be found in the file LICENSE.md at
+// the top level directory of deal.II.
+//
+// ---------------------------------------------------------------------
+
+#ifndef dealii_solver_idr_h
+#define dealii_solver_idr_h
+
+
+#include <deal.II/base/config.h>
+
+#include <deal.II/base/logstream.h>
+#include <deal.II/base/signaling_nan.h>
+#include <deal.II/base/subscriptor.h>
+#include <deal.II/base/utilities.h>
+
+#include <deal.II/lac/full_matrix.h>
+#include <deal.II/lac/solver.h>
+#include <deal.II/lac/solver_control.h>
+
+#include <boost/random.hpp>
```

Comment:
Almost: You need to change the header file:
```suggestion
#include <random>
```

Reply:
Yes, of course!